config.json

```diff
@@ -376,6 +376,19 @@
       "interviewer_handle": "jonathandmiddleton",
       "interviewee_handle": "taiyab",
       "youtube_id": "HBZpD1K9saE"
+    },
+    {
+      "uuid": "363b6eeb-19b6-4270-90ad-3cf3de21d32d",
+      "slug": "prolific-mentor-and-system-automater",
+      "title": "Prolific Mentor and System Automator",
+      "blurb": "Negatively labelled a jack-of-all-trades? Maybe it's actually a good thing, especially when mentoring people from diverse backgrounds, programming experience and cultures! Isaac is a self-proclaimed techie and Jack of all trades...amongst many other things!",
+      "published_at": "2022-16-09 17:01",
```

Comment:
This is a date in the past, so if we merge this, the story will immediately be published. This is likely not what you want, correct?